src/main/java/org/springframework/social/bitbucket/api/BitBucketUser.java

```diff
@@ -41,7 +41,7 @@
     private String lastName;
 
     @JsonProperty("is_team") @Getter
-    private boolean team;
+    private Boolean isTeam;
```

Comment:
is_team could be null if data was fetched by unauthorized user. I'll place that comment in field's javadoc.